cse-cc-export-runner-app/src/main/java/com/farao_community/farao/cse/export_runner/app/services/FileImporter.java

```diff
@@ -42,17 +45,29 @@ Network importNetwork(String cgmUrl) {
         return Network.read(FileUtil.getFilenameFromUrl(cgmUrl), openUrlStream(cgmUrl));
     }
 
-    CseCrac importCseCrac(String cracUrl) {
+    CRACDocumentType importCseCrac(String cracUrl) {
         InputStream cracInputStream = openUrlStream(cracUrl);
-        CseCracImporter cseCracImporter = new CseCracImporter();
-        return cseCracImporter.importNativeCrac(cracInputStream);
+        return importNativeCrac(cracInputStream);
+    }
+
+    CRACDocumentType importNativeCrac(InputStream inputStream) {
```

Comment:
This method looks like it should be extracted to a common utility class: such a treatment is also declared and used in the `FileImporter` from the import_runner and in the `BusBarChangePreProcessor`.

Reply:
As discussed, this point should be handled in a separate ticket dedicated to refactoring. Won't be changed in this PR.